test-case/check-userspace-paplay.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+
+##
+## Case Name: check-userspace-paplay
+## Preconditions:
+##    N/A
+## Description:
+##    Go through all the sinks and paplay on the sinks whose
+##    active port is available (jack connected) or unknown (speaker)
+## Case step:
+##    1. go through all the sinks
+##    2. check the sink's active port is available or not
+##    3. paplay on the active port which is available or unknown
+## Expect result:
+##    paplay on the sinks successfully
+##
+
+# shellcheck source=case-lib/lib.sh
+source "$(dirname "${BASH_SOURCE[0]}")"/../case-lib/lib.sh
+
+# check pulseaudio runs properly or not
+if ! func_lib_check_pa; then
+    die "Please check whether pulseaudio runs correctly or not"
```

Comment:
Avoid negations when not needed, see above. Shorter, clearer and looking more like an assert and plain English and like most other shell code:
```shell
func_lib_check_pa || die ...
```